pkg/operators/machine-api-operator.go

```diff
@@ -14,7 +14,7 @@ var (
 	maoManagedDeployment = "machine-api-controllers"
 )
 
-var _ = Describe("[Feature:Operators] Machine API operator deployment should", func() {
+var _ = Describe("[Feature:Operators] Machine API operator deployment should [Serial]", func() {
```

Comment:
Does naming these as `[serial]` make them run as serial?

Reply:
not, specifically. these are labels inspired from the kubernetes community[0]

[0] https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md#kinds-of-tests